cime_config/tests.py

```diff
@@ -110,6 +110,7 @@
         "tests" : (
             "ERS_P480_Ld5.T62_oEC60to30v3wLI.GMPAS-DIB-IAF-ISMF",
             "PEM_P480_Ld5.T62_oEC60to30v3wLI.GMPAS-DIB-IAF-ISMF",
+            "SMS.ne30_oECv3_gis.IGCLM45_MLI.clm-extrasnowlayers",
```

Comment:
Should the compset use `ELM` instead of `CLM45`?

Reply:
Thanks @bishtgautam - yes, that's absolutely correct. I had another set of commits for this PR but decided it was suffering from PR-creep, so I hadn't checked all the changes in that last commit. But this was one of them... I'll fix that and commit right now